.travis.yml

```diff
@@ -34,6 +34,7 @@ cache:
 
 before_install:
   - composer global require satooshi/php-coveralls:@stable sllh/composer-lint:@stable --no-update
+  - composer global require phpunit/phpunit:@stable
```

Comment:
I do not understand, why not just add `phpunit/phpunit:@stable` on the already present `global require` line?

Reply:
Yeah, i agree @soullivaneuh
Just adding on last position of that line.